<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="StartButton">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="12" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Priority (1-5)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" rowspan="14">
         <widget class="QListWidget" name="PendingList"/>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="PendingLabel">
          <property name="text">
           <string>Pending</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QPushButton" name="AddButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="UndoButton">
          <property name="text">
           <string>Undo</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="RedoButton">
          <property name="text">
           <string>Redo</string>
          </property>
         </widget>
        </item>
        <item row="15" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Due Date(dd-mm-yy)</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Description</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="ReloadButton">
          <property name="text">
           <string>Reload</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="CompleteLabel">
          <property name="text">
           <string>Complete</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3" rowspan="14">
         <widget class="QListWidget" name="CompleteList"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Task</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="SearchButton">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="InProgressLabel">
          <property name="text">
           <string>In Progress</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLineEdit" name="DueDateLineEdit"/>
        </item>
        <item row="6" column="0">
         <widget class="QLineEdit" name="DescriptionLineEdit"/>
        </item>
        <item row="4" column="0">
         <widget class="QLineEdit" name="TaskLineEdit"/>
        </item>
        <item row="3" column="2" rowspan="14">
         <widget class="QListWidget" name="InProgressList"/>
        </item>
        <item row="10" column="0">
         <widget class="QLineEdit" name="PriorityLineEdit"/>
        </item>
        <item row="13" column="0">
         <widget class="QPushButton" name="SortByDeadlineButton">
          <property name="text">
           <string>Sort By Deadline</string>
          </property>
         </widget>
        </item>
        <item row="14" column="0">
         <widget class="QPushButton" name="SortByPriorityButton">
          <property name="text">
           <string>Sort By Priorty</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="NotificationButton">
          <property name="text">
           <string>Notification</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
